source/PlayerInfo.h

```diff
@@ -180,6 +180,17 @@ class PlayerInfo {
 	
 	// Access the "condition" flags for this player.
 	int64_t GetCondition(const std::string &name) const;
+	// Copy conditions matching the prefix into the given map.
+	void GetConditions(std::map<std::string, int64_t> &targetMap, const std::string &prefix) const;
```

Comment:
Why modify a map handed to the function instead of returning a map? Are we expecting to have a situation where we want to add to an existing map in a function calling GetConditions?

Reply:
Yes, I expect some situations in the future where the map gets filled by recursive calls. Those calls can be implemented by a map that is returned, which will be more readable, or with the current implementation, which should have a somewhat better performance.

The code here is not performance critical, so if we prefer readability, then I can change this to a function that returns a map.

Reply:
I'll leave it up to tehhowch. I think it's readable enough both ways, but having the function return a map is probably more readable than the current method of modifying a map.

Reply:
Will be replaced by iterator (maybe filtered iterator) as per discussion with tehhowch.